dolphinscheduler-ui/src/js/conf/login/index.js

```diff
@@ -18,6 +18,7 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
 // import $ from 'jquery'
+import 'babel-polyfill'
 import Vue from 'vue'
 import App from './App'
 import i18n from '@/module/i18n'
```

Comment:
Keep spaces

Reply:
It's just ok, I don't know why it turns red